enhancements/baremetal/baremetal-keepalived-unicast.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Baremetal-Keepalived-Unicast
+authors:
+  - "@bcrochet"
+  - "@yboaron"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-02-21
+last-updated: 2020-02-21
+status: implementable
+---
+
+# baremetal-keepalived-unicast
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Non cloud platforms use VIPs (Virtual IP) and [Keepalived](https://www.keepalived.org/) to support high availability for API and default Ingress.
+For more information in the matter of VIPs please use the following [link](https://github.com/openshift/enhancements/blob/master/enhancements/network/baremetal-networking.md).
+
+In current implementation, Keepalived is configured to run [VRRP management protocol](https://tools.ietf.org/html/rfc3768#section-3) of the VIP addresses in a multicast mode, but since not all environments have multicast available, this proposal seeks to remedy that.
```

Comment:
what i would like the summary to include is whether we will be replacing with new impl across the board or are we providing an alternate way for users to choose.

Reply:
Done